imports/plugins/included/search-mongo/server/methods/searchcollections.js

```diff
@@ -185,8 +185,8 @@ export function buildOrderSearchRecord(orderId) {
 
   orderSearch.billingName = shopBilling.address && shopBilling.address.fullName;
   orderSearch.billingPhone = _.replace(shopBilling.address && shopBilling.address.phone, /\D/g, "");
-  orderSearch.shippingName = shopShipping.address.fullName;
-  orderSearch.shippingPhone = _.replace(shopShipping.address.phone, /\D/g, "");
+  orderSearch.shippingName = shopShipping.address && shopShipping.address.fullName;
+  orderSearch.shippingPhone = _.replace(shopShipping.address && shopShipping.address.phone, /\D/g, "");
```

Comment:
Any reason we can't use prototype `replace` here? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace